docs/management/admin/blocklist.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[blocklist]]
+[chapter]
+= Blocklist
+
+coming[8.2.0]
+
+The blocklist allows you to prevent specified applications from running on hosts, extending the list of processes that {endpoint-sec} considers malicious. This is especially useful to ensure that known malicious processes aren't accidentally executed by an end user. 
+
+[NOTE] 
+=====
+In addition to configuring specific entries on the **Blocklist** page, you must also ensure that the blocklist is enabled on the {endpoint-sec} integration policy in the <<malware-protection, Malware protection settings>>. This setting is enabled by default.
+
+You must have the built-in `superuser` role to access the blocklist. For more information, refer to {ref}/built-in-users.html[Built-in users].
```

Comment:
Since most of the pages for endpoint management are so similar, I'd like to make this update in a separate issue that includes adding the prerequisites box consistently across all the pages. I was already thinking of creating an issue for general consistency fixes, so I think the prereqs box would fall nicely into that issue's scope. I'll add a link to this PR once I've created the other issue.